packages/nestjs-client/src/connections/connection.entity.ts

```diff
@@ -14,7 +14,7 @@ export class ConnectionEntity {
   status?: ExtendedDidExchangeState
 
   @Column({ type: 'varchar', nullable: true })
-  language?: string
+  lang?: string
 
```

Comment:
More on this: what about creating a specific field for features, which will contain an object with all capabilities, protocols, etc. discovered? This could make nestjs-client a bit more friendly to manage.